packages/devextreme/scss/widgets/generic/tabs/variables/_sizes.scss

```diff
@@ -24,8 +23,8 @@ $generic-badge-min-size: null !default;
   $generic-tab-text-line-height: 18px !default;
   $generic-tabs-nav-button-height: 38px !default;
   $generic-tabs-icon-margin: 8px !default;
-  $generic-tabs-icon-size: 18px !default;
   $generic-badge-min-size: 18px !default;
+  $generic-tabs-icon-size: 18px !default;
```

Comment:
The sizes of the icons in the Tabs match the sizes of the basic icon values. I suggest not duplicating these values. These values are already included in the basic variables $generic-base-icon-size, $material-base-icon-size, $fluent-base-icon-size.

Reply:
[[Tabs: Use base constants for icon width](https://github.com/DevExpress/DevExtreme/pull/26037)]